flow360/component/simulation/meshing_param/params.py

```diff
@@ -18,23 +19,58 @@ class MeshingParameters(Flow360BaseModel):
-    gap_treatment_strength: Optional[float]
-        Narrow gap treatment strength used when two surfaces are in close proximity. Use a value between 0 and 1, where 0 is no treatment and 1 is the most conservative treatment. This parameter has a global impact where the anisotropic transition into the isotropic mesh. However, the impact on regions without close proximity is negligible.
-    refinements: Optional[List[Union[EdgeRefinementTypes, FaceRefinement, ZoneRefinementTypes]]]
-        Refinements for meshing.
+    Affects volume meshing:
+    - farfield
+    - refinement_factor
+    - gap_treatment_strength
+    - volume_layer_growth_rate
+    - volume_layer_first_layer_thickness
+    - `class` UniformRefinement
+    - `class` CylindricalRefinement
+
+    Affects surface meshing:
+    - max_edge_length
+    - curvature_resolution_angle
+    - surface_layer_growth_rate
+    - `class` FaceRefinement
+    - `class` AnisoSurfaceEdge
+    - `class` ProjectAnisoSurfaceEdge
     """
 
-    # Global fields:
-    farfield: Optional[Farfield] = pd.Field()
-    refinement_factor: Optional[pd.PositiveFloat] = pd.Field()
-    gap_treatment_strength: Optional[float] = pd.Field(ge=0, le=1)
+    # Volume **defaults**:
+    farfield: Literal["auto", "quasi-3d", "user-defined"] = pd.Field(
+        description="Type of farfield generation."
+    )
+    refinement_factor: pd.PositiveFloat = pd.Field(
+        description="If refinementFactor=r is provided all spacings in refinement regions and first layer thickness will be adjusted to generate r-times finer mesh."
+    )
+    gap_treatment_strength: float = pd.Field(
+        ge=0,
+        le=1,
+        description="Narrow gap treatment strength used when two surfaces are in close proximity. Use a value between 0 and 1, where 0 is no treatment and 1 is the most conservative treatment. This parameter has a global impact where the anisotropic transition into the isotropic mesh. However, the impact on regions without close proximity is negligible.",
+    )
+    volume_layer_growth_rate: pd.PositiveFloat = pd.Field(
+        gt=1, description="Growth rate for volume prism layers."
+    )
+    volume_layer_first_layer_thickness: pd.PositiveFloat = pd.Field(
+        description="Default first layer thickness for volumetric anisotropic layers."
+    )
+
+    # Face **defaults**
+    max_edge_length: LengthType.PositiveFloat = pd.Field(
+        description="Global maximum edge length for surface cells. This value will be overwritten by the local specification if provided."
+    )  # Note: Applied to faces without faceName and therefore has to exist in root level.
+    curvature_resolution_angle: pd.PositiveFloat = pd.Field(
+        description="""
+        Global maximum angular deviation in degrees. This value will restrict:
+        (1) The angle between a cell’s normal and its underlying surface normal
+        (2) The angle between a line segment’s normal and its underlying curve normal
+        """
+    )
```

Comment:
Similarly, I think this is not needed. It can be achieved by:
```python
MeshingParameters(refinements=[FaceRefinement(entities="*", max_edge_length=...,  curvature_resolution_angle=...)]
```

Reply:
`entities="*"` will not capture faces that has no name. faceName is not a required property as of now. @rundaji showed me some customer does not always provide faceName so it is not possible to select all faces using `entities`/metadata.

Reply:
You are talking about implementation detail. We can convert "*" to global parameter as we have now. I don't understand why we need to expand "*". It is a global setting and is compatible with our current production mesher. It doesn't matter for the user. What only changes is input format.

I just don't like having 2 things achieving the same goal. The other thing is that we need to name all entities, if something is not named in CSM script, we will just use "unnamed" for it.

Reply:
Then there is a logic inconsistency right?
If the user do: 
```python
entities = [my_geometry["wing*], "*"]
```
it means wings plus all hidden faces. But if user do:
``python
entities = "*"
```
then it is for every thing?

Maybe an alternative is to ask user skip entities specification. Then we assume it is a global one. This is less logically in conflict with the "*" one.

Reply:
As discussed, we omit "entities" to specify global properties.